cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -516,6 +516,15 @@
   </property>
 
   <property>
+    <name>program.status.event.topic</name>
```

Comment:
Doc build will fail once the PR is merged, you will need to update the md5hash then